assets/js/blocks/cart-checkout/cart/frontend.js

```diff
@@ -32,12 +35,16 @@ const Block = ( { emptyCart, attributes } ) => {
 				<RawHTML>{ emptyCart }</RawHTML>
 			) : (
 				<LoadingMask showSpinner={ true } isLoading={ cartIsLoading }>
-					<FullCart
-						isShippingCalculatorEnabled={
-							attributes.isShippingCalculatorEnabled
-						}
-						isShippingCostHidden={ attributes.isShippingCostHidden }
-					/>
+					<ValidationContextProvider>
```

Comment:
I wonder if we should somehow link this context provider to our `<form>` components via a new component that implements it. I'm assuming validation and forms go hand in hand.

Reply:
I like the idea and it makes sense, however that would prevent the checkout context (for processing checkout) to be aware of validation. So to implement this we'd probably have to move validation to a global store.  For now, I'd like to roll with this because it has flexibility across the Cart (which doesn't have checkout context) and checkout (which does have checkout context).  

We can look at iterating on this to make it more form specific (I'd like to get a better idea of how payment methods might implement validation and make checkout aware of that before committing to another approach).